### PR TITLE
fix: proposition 3 of integrated polynomial

### LINKED ISSUE #7: deg(R) = d => R = 0 ? Should be deg(R) = -∞, right ?
Here is the highlighted version of the proof :

SCR-20240118-lffs

This issue is to be sure if it's really a problem or just me not understanding 💀 !

### PR DESCRIPTION
**Pull request by GabinDDL**
# fix: proposition 3 of integrated polynomial

## Description

There was a problem in the proof of the polynomial ring is integral, implies that the ring is integral, solved in this MR. The problem was due to an inconsistency in the degree of the remainder in the proof

## Changes Made

The proof of proposition 3 of the section "Rings of polynomials" has been corrected.

## Related Issue 

#7 

## Checklist

- [x] I have self-reviewed my code
- [x] Code follows project's style guidelines
- [x] Tests added and passing
- [x] Documentation updated (if needed)

## @Mention 

@mathusanMe @Yag000

### CODE REVIEW

File: algebre-II-notes.tex [hunk 1]
```diff
@@ -96,8 +96,8 @@ \subsection{Division euclidienne}
 	$\mathcal{P}$ est non vide et minoré par $0$. Donc il existe $d \in \mathcal{P}$ tel que $d$ est minimal.
 	On note $Q$ un polynôme de $I$ de degré $d$. Soit $P \in I$. On réalise la division euclidienne de $P$ par $Q$:
 	\[ \exists (U, R) \in K[X] \times K[X] \mid P = UQ + R \text{ et } \deg(R) < \deg(Q) \]
-	On a que $R = P - UQ \in I$ et donc $\deg(R) \in \mathcal{P}$. Donc $\deg(R) \geq d$ par définition de $d$.
-	Donc $\deg(R) = d$ et donc $R = 0$ car $d$ est minimal. Donc $P = UQ$ et donc $I = <Q>$.
+	On a que $R = P - UQ \in I$  car $I$ est un idéal, et donc $\deg(R) < d$. Par définition de $d$,
+	on a $R = 0$ car $d$ est minimal. Donc $P = UQ$ et donc $I = <Q>$.
```
Comment (Yag000) [inline algebre-II-notes.tex]:
```suggestion
	On a que $R = P - UQ \in I$  car $I$ est un idéal. Comme $\deg(R) < d$ et par définition $d$ est minimal pour tous les éléments non nuls, on a que $R = 0$.  Donc $P = UQ$ et donc $I = <Q>$.
```

**Review by Yag000 — CHANGES_REQUESTED** [review]

File: algebre-II-notes.tex (hunk 1 above)
Comment (GabinDDL) [reply algebre-II-notes.tex]: resolved

File: algebre-II-notes.tex [hunk 2]
```diff
@@ -96,8 +96,8 @@ \subsection{Division euclidienne}
 	$\mathcal{P}$ est non vide et minoré par $0$. Donc il existe $d \in \mathcal{P}$ tel que $d$ est minimal.
 	On note $Q$ un polynôme de $I$ de degré $d$. Soit $P \in I$. On réalise la division euclidienne de $P$ par $Q$:
 	\[ \exists (U, R) \in K[X] \times K[X] \mid P = UQ + R \text{ et } \deg(R) < \deg(Q) \]
-	On a que $R = P - UQ \in I$ et donc $\deg(R) \in \mathcal{P}$. Donc $\deg(R) \geq d$ par définition de $d$.
-	Donc $\deg(R) = d$ et donc $R = 0$ car $d$ est minimal. Donc $P = UQ$ et donc $I = <Q>$.
+	On a que $R = P - UQ \in I$  car $I$ est un idéal. Comme $\deg(R) < d$, par définition de $d$,
+	on a que $R = 0$ car $d$ est minimal. Donc $P = UQ$ et donc $I = <Q>$.
```
Comment (Yag000) [inline algebre-II-notes.tex]:
```suggestion
On a que $R = P - UQ \in I$  car $I$ est un idéal. Comme $\deg(R) < d$ et par définition $d$ est minimal pour tous
 les éléments non nuls, on a que $R = 0$.  Donc $P = UQ$ et donc $I = <Q>$.
```

**Review by Yag000 — CHANGES_REQUESTED** [review]

File: algebre-II-notes.tex (hunk 1 above)
Comment (Yag000) [reply algebre-II-notes.tex]: You responded too quickly, I just changed the suggestion: https://github.com/Yag000/algebre-II-notes/pull/8#discussion_r1457718094

File: algebre-II-notes.tex (hunk 2 above)
Comment (GabinDDL) [reply algebre-II-notes.tex]: resolved too

**Review by Yag000 — APPROVED** [review]

**Pull request merged**